test/e2e/traceflow_test.go

```diff
@@ -103,6 +103,8 @@ func testTraceflowIntraNodeANP(t *testing.T, data *TestData) {
 	node1 := nodeName(nodeIdx)
 	node1Pods, _, node1CleanupFn := createTestAgnhostPods(t, data, 3, data.testNamespace, node1)
 	defer node1CleanupFn()
+	// Give a little time for Nodes to install OVS flows.
+	time.Sleep(time.Second * 2)
```

Comment:
Why is this added? Is this related with containerd change?

Reply:
Yes. Containerd use connectInterfaceToOVSAsync to create port asynchronously, which could cause execution time difference compared to docker. In my test, waiting for 1s is enough for the test to pass.

Reply:
You may refine the comment with more details like `Containerd creates port asynchronously, which could cause execution time longer compared to docker.`.

Reply:
Updated, thanks